src/exp/kafka-watcher/cmd/main.go

```diff
@@ -38,6 +38,12 @@ func main() {
 	if err != nil {
 		logrus.WithError(err).Panic()
 	}
+	if len(kafkaServers) == 0 {
```

Comment:
I think it shouldn't panic in this case, let's print an error instead? The mapper does more than just watch Kafka, so it's not desirable for it to fail to start in this case.

Also, use logrus.WithFields to log the kafka servers field instead of Errorf